proposals/0000-template.md

```diff
@@ -83,6 +83,37 @@ learnability of the language for novice users. Define and list any remaining
 drawbacks that cannot be resolved.
 
 
+## Backward Compatibility
+
+Will your proposed change cause any existing programs to change behaviour or
+stop working? Assess the expected impact on existing code on the following scale:
```

Comment:
Can we point to the furtehr specifics info in the template.md file?  Or reproduce those specifics here if you prefer

Reply:
I've added a reference in the README.